### ISSUE TITLE
"Add Server" button is missing

### ISSUE THREAD

**Issue by xuesoso** [rstudio/rsconnect-jupyter#362]
Running on a fresh installation of jupyter notebook on "Ubuntu 20.04.2 LTS". 

When setting up a new server, the buttons for "Add Server" and "Cancel" are missing. Any suggestions on how to get the buttons back?

package version:
```
jupyter_client                    8.1.0
jupyter-contrib-core              0.4.2
jupyter-contrib-nbextensions      0.7.0
jupyter_core                      5.3.0
jupyter-highlight-selected-word   0.2.0
jupyter-nbextensions-configurator 0.6.1
jupyter-server                    1.23.4
jupyterlab-pygments               0.1.2
rsconnect-jupyter                 1.8.0
rsconnect-python                 1.16.0
nbconvert                         6.5.4
```
Screenshot of the buggy version:
<img width="400" alt="image" src="https://user-images.githubusercontent.com/15006047/231289592-6ad3d6c0-b3c9-4c3f-b0c9-1c8809ded71f.png">

What it should look like (on `rsconnect-jupyter==1.4.5` and `rsconnect-python==1.7.0`):
<img width="400" alt="image" src="https://user-images.githubusercontent.com/15006047/231289743-837b8932-cfa9-40f0-8807-a9a0c0f912bf.png">

**Comment by bcwu** [comment]
Hi @xuesoso, can we get a bit more information on how the plugin was installed? Was it installed thorugh the steps in our [documentation](https://docs.posit.co/rsconnect-jupyter/)? Or was it cloned from the repository and then installed?

**Comment by xuesoso** [comment]
that's right. I followed the documentation and tried installing both via `pip` or `conda`, but got similar results.

**Comment by bcwu** [comment]
hi @xuesoso we haven't been able to run this one down. Could you tell us which Python version you were using, what is in the requirements.txt or environment.yml when you were doing the install for pip and conda? Thanks.
